contents/blog/growth-loops.md

```diff
@@ -0,0 +1,109 @@
+
+For example, with Google Ad Words, companies pay per click on specific keywords related to potential customers’ searches. They pay for clicks (performance). With the correct math figured out (cost of acquisition), this can lead to a growth loop of increasing ad spend and revenue growth.
+
+Products that benefit from performance marketing are generally high margins (more money for ads) with low organic demand. For example, direct-to-consumer goods (e.g. Casper, Warby Parker, Dollar Shave Club) and SaaS software targeting small-medium businesses (e.g. Shopify, Hubspot, Slack).
+
+### Content
+
+Content is the growth loop of users creating content that other users find which attracts them to become users (and create content). It is also the ability to generate data used to attract users who then create more data. 
+
+Two types of products benefit from this:
+
+- Users discover your product through content other users create and share (e.g. StackOverflow, Reddit, Substack). This is also known as user-generated content.
+- Your product generates a large amount of data, which you use to make content (e.g. TripAdvisor, Zillow, Expedia). These companies rely heavily on SEO.
+
+### Sales
+
+Sales is the growth loop of acquiring users through sales. Customers grow (both revenue and usage), which allows the company to hire more salespeople as well as improve the product’s value proposition.
+
+Products relying on sales are generally large contracts that need high levels of support, engagement, and customization. Offers are often tailored to the specific customer and require a large amount of time and effort. 
+
+For example, sales is a growth loop for enterprise SaaS companies (e.g. Salesforce, SAP, Microsoft) and big consultancies (e.g. McKinsey, Deloitte)
+
+## Choosing the right growth loop
+
+After reading these, you likely have some idea of what growth loop is right for your product and company. Two key questions to understand this further:
+
+1. Who are your ideal users?
+2. What benefits do those users get as part of your growth loop?
+
+Understanding your ideal user will help you decide where to put your effort for your growth loop. If your ideal user won’t pay for your product, it isn’t a good idea to use sales as your growth loop. You can create a growth loop that better aligns with what they want. Growth loops are about creating win-wins.
+
+A user needs a win-win to repeatedly take an action that fuels a product’s growth loop. They need an incentive. Smart companies build this incentive into the core of their product. For example, you gain followers by posting on social media, but that also keeps users engaged and attracts new users.
+
+The benefits your product and growth loop can provide are limitless. Finding new ways to provide this value is a great way.
```

Comment:
Again, I think this section would benefit from one or two examples. Walk people through the process of choosing a growth loop, maybe using some well-known companies as theoretical examples, or just different types of companies (e.g. one B2B example, one B2C example)

Reply:
Added!